lib/process-services-cloud/src/lib/people/mock/custom-people-cloud-mock.service.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { PeopleCloudServiceInterface } from '../../services/people-cloud-service.interface';
+import { Observable, of } from 'rxjs';
+import { IdentityUserModel } from 'core';
+
+export const customServiceMockUsers = [
+    { id: 'custom-fake-id-1', username: 'custom first-name-1 last-name-1', firstName: 'custom-first-name-1', lastName: 'custom-last-name-1', email: 'custom-abc@xyz.com' },
+    { id: 'custom-fake-id-2', username: 'custom first-name-2 last-name-2', firstName: 'custom-first-name-2', lastName: 'custom-last-name-2', email: 'custom-abcd@xyz.com'}
+];
+
+@Injectable()
+export class CustomMockPeopleCloudService implements PeopleCloudServiceInterface {
+    findUsers(_searchTerm: string): Observable<IdentityUserModel[]> {
```

Comment:
why _ as prefix?

Reply:
![image](https://user-images.githubusercontent.com/25585550/96999333-a1252600-1552-11eb-8d83-6a8403060016.png)
To avoid above warning, tslint suggested to keep _ as prefix

Reply:
let's check the proper way to skip the check

Reply:
@mauriziovitale I think this is ok to skip the check, it's the linter convention for the cases when parameter is not used but should be present